build.psm1

```diff
@@ -654,9 +609,13 @@ function New-PSOptions {
     if (-not $Configuration) {
         $Configuration = 'Debug'
     }
-
     Write-Verbose "Using configuration '$Configuration'"
 
+    if (-not $Framework) {
+        $Framework = "netcoreapp3.0"
```

Comment:
It would be nice to have the constant in one place. Maybe read from PowerShell.Coom. props file?

Reply:
I can have this read from PS.Common.Props, but other locations aren't using that file